README.md

```diff
@@ -212,7 +212,7 @@ For [OpenSUSE, CentOS, Fedora, RHEL and SLE](https://build.opensuse.org/package/
 Kai-Uwe Behrmann kindly provides packages [for download](http://software.opensuse.org/download.html?project=home%3Abekun%3Adevel&package=libelektra4).
 For Debian wheezy and jessie amd64 we provide latest builds. See build server below.
 
-If there are no packages available for your distribution, see the
+If there are no packages available for your distribution, or if you are using a different operating system like macOS, see the
```

Comment:
macOS now has homebrew packages, I think we can recommend that now.

Please also try it and give feedback.